egs/wsj/s5/steps/nnet3/lstm/train_perturbed.sh

```diff
@@ -0,0 +1,732 @@
+#!/bin/bash
```

Comment:
It would be better to just make perturbed training an option in the original lstm script.